persistence-cassandra-4.0/src/main/java/io/stargate/db/cassandra/impl/StargateQueryHandler.java

```diff
@@ -94,6 +95,7 @@
   private static final Logger logger = LoggerFactory.getLogger(StargateQueryHandler.class);
   private final List<QueryInterceptor> interceptors = new CopyOnWriteArrayList<>();
   private AtomicReference<AuthorizationService> authorizationService;
+  private static final Map<String, String> EMPTY_HEADERS = Collections.emptyMap();
```

Comment:
Same comment as 3.11. We should be passing the properties/headers in all cases.

Reply:
I don't understand this comment. What do you mean? Do you mean that we should rename it?

Reply:
https://github.com/stargate/stargate/pull/581#discussion_r560285601